packages/react/package.json

```
@@ -57,9 +57,9 @@
   "dependencies": {
     "@aws-amplify/ui": "6.7.2",
     "@aws-amplify/ui-react-core": "3.1.2",
-    "@radix-ui/react-direction": "^1.1.0",
-    "@radix-ui/react-dropdown-menu": "^2.1.0",
-    "@radix-ui/react-slider": "^1.2.0",
+    "@radix-ui/react-direction": "1.0.0",
+    "@radix-ui/react-dropdown-menu": "1.0.0",
+    "@radix-ui/react-slider": "1.0.0",
```

Although this PR pins dependencies, it also downgrades three dependencies to some pretty old releases. If you want to avoid unexpected version bumps downstream, pinning is fine, but it's best to use the latest available versions:

```suggestion
    "@radix-ui/react-direction": "1.1.0",
    "@radix-ui/react-dropdown-menu": "2.1.4",
    "@radix-ui/react-slider": "1.2.2",
```